### DISCUSSION TITLE
Why is Mobx not tracking observable value passed from constructor parameter?

### DISCUSSION THREAD

**Discussion by sophon-aniketos** [mobxjs/mobx#3130] [Q&A] [answered]
Hi guys,

I'm new to Mobx but loving this library so far. I have this question. I hope someone can answer.

https://stackoverflow.com/questions/69418791/mobx-reaction-not-tracking-observable-value-passed-from-constructor-parameter

**Comment by mweststrate** [comment, marked as answer]
Seems question was answered already
